diff -urN aa/evolution-webcal-1.0.2/evolution-webcal.applications.in evolution-webcal-1.0.2/evolution-webcal.applications.in
--- aa/evolution-webcal-1.0.2/evolution-webcal.applications.in	2004-02-07 23:20:59.000000000 +0100
+++ evolution-webcal-1.0.2/evolution-webcal.applications.in	2004-03-18 16:03:42.583818960 +0100
@@ -1,5 +1,5 @@
 evolution-icalshare
-	command=@libexecdir@/evolution-icalshare
+	command=@libexecdir@/evolution-webcal
 	name=Evolution iCalendar Subscription Interface
 	can_open_multiple_files=true
 	expects_uris=true
