diff -urN evolution-webcal-1.0.9.orig/configure.ac evolution-webcal-1.0.9/configure.ac
--- evolution-webcal-1.0.9.orig/configure.ac	2004-08-16 22:49:49.805062192 +0200
+++ evolution-webcal-1.0.9/configure.ac	2004-08-16 22:50:14.271342752 +0200
@@ -18,7 +18,7 @@
 	[translation domain of this package])
 AC_SUBST(GETTEXT_PACKAGE)
 
-ALL_LINGUAS="ar az bg bn bs ca cs da de el en_CA en_GB es eu fi gu hu ja ko ms nb nl no pa pl pt pt_BR sq sr sr@Latn sv ta uk"
+ALL_LINGUAS="ar az bg bn bs ca cs da de el en_CA en_GB es eu fi gu hu ja ko ms nb nl pa pl pt pt_BR sq sr sr@Latn sv ta uk"
 AM_GLIB_GNU_GETTEXT
 
 # AM_GLIB_GNU_GETTEXT above subst. $DATADIRNAME
